persistent/tests/test_timestamp.py

```diff
@@ -198,14 +199,74 @@ def check_py3(op, passes):
                 check(op, passes)
 
 
-class TimeStampTests(pyTimeStampTests):
+class pyTimeStampTests(TimeStampTests):
+    # Things specific to the Python implementation
```

Comment:
It would be nice to verify that the C code still returns the same hash values for the same timestamps.  (Granted, the old code didn't do that.)

The monkey-patching parts to force a 32- or 64-bit algorithm from the Python implementation is, of course, specific to the Python implementation.